File: src/main/java/de/maggiwuerze/xdccloader/persistency/ServerRepository.java
package de.maggiwuerze.xdccloader.persistency;

import de.maggiwuerze.xdccloader.model.entity.Server;
import java.util.List;
import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface ServerRepository extends PagingAndSortingRepository<Server, Long> {

	List<Server> findAll();
}

File: src/main/kotlin/de/maggiwuerze/xdccloader/persistency/ServerRepository.kt
package de.maggiwuerze.xdccloader.persistency

import de.maggiwuerze.xdccloader.model.Server
import de.maggiwuerze.xdccloader.model.User
import org.springframework.data.repository.CrudRepository
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.stereotype.Repository
import java.util.*


@Repository
interface ServerRepository : PagingAndSortingRepository<Server, Long> {

}